### ISSUE TITLE
Module installer prompts "Installing may cause problems" but provides no explanation

### ISSUE THREAD

**Issue by teppokoivula** [processwire/processwire-issues#188]
While attempting to install a module, in this case RepeaterMatrix, the installer outputs a JavaScript confirm box stating only that "Installing may cause problems. Are you sure you want to install?". 

Apparently this happens if the module has unfulfilled requirements, like when installing a "multipart module", which is intended to install it's own requirements. The problem, in my opinion, is that this message is way too vague for the user to make any informed choices based on it -- at least without diving into the core code in order to figure out what's the actual risk involved.

If this (unfulfilled requirements) is indeed the only situation where this message can occur, I think it should clearly state that *module requirements are currently not fulfilled*, and preferably also name *which* requirements.

I also don't see much point in this message in case that the module is configured to install it's own requirements and they have been found from the modules directory. It signals that there's something wrong with the module, while this is a completely normal use case.

And, just to be clear, I can deal with this problem, but if we're thinking in terms of a wider audience, I feel that the current behaviour is pretty confusing. :)

* ProcessWire version: 3.0.42

**Comment by teppokoivula** [comment]
Taking a closer look at this, it seems that I misunderstood part of what's happening here:

- Aforementioned notice is only displayed when I'm trying to install a module like RepeaterMatrix "the wrong way", in this case via the Inputfield instead of the Fieldtype. This particular module won't install properly that way, so the alert makes sense (one shouldn't attempt to install it this way), but I still feel that it's not clear *why* that would be.

- This is somewhat unrelated, but when I set up a new site and attempt to install RepeaterMatrix via the Fieldtype, I got the same error there. What I missed earlier is that *it's not for the same reason*: FieldtypeRepeaterMatrix requires FieldtypeRepeater, and even though it will apparently install it automatically, a JavaScript warning is still displayed in the modules list, and an additional system error is displayed after installing FieldtypeRepeaterMatrix.

Sorry for the confusion, but to summarise: I still feel that this particular message is too vague to be really useful. I guess one might even argue that my confusion over this issue serves to prove that :)
